wordpress-mobile/WordPress-iOS
# Site Creation: Verticals screen return button changed to "Done".

This PR adds something that [was supposed to be included in my previous PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/12071#issuecomment-510158422) but wasn't.

The return button in the Verticals screen now reads "Done".

## Testing:

1. Create a new WP.com site.
2. In the verticals screen (step 1), tap on the field.
3. Make sure the vertical keyboard has "Done" in place of the return button.

## Update release notes:
